app/src/main/java/com/obrasocialsjd/magicline/activities/main/general/MainActivity.kt

```diff
@@ -158,6 +166,37 @@ class MainActivity : BaseActivity() {
         }
     }
 
+    private fun getNotificationNews(){
+        myNewsAdapter = NewsAdapter(ArrayList())
+
+        mMagicLineViewModel.getPosts(getAPILang(context = applicationContext)).observe(this, androidx.lifecycle.Observer {
+            myNewsAdapter.loadItems(toNewsModel(it))
+            myNewsAdapter.notifyDataSetChanged()})
+    }
+
+    private fun toNewsModel(list: List<PostsItem>): List<NewsModel> {
+        val news : MutableList<NewsModel> = mutableListOf()
+        val onClickListener: (NewsModel) -> Unit = {}
+
+            val detailModel = DetailModel(
```

Comment:
if index is always 0,give a descriptive name and move to Constants.kt. Else review